serveradmin/settings.py

```diff
@@ -217,6 +217,21 @@
     'graphOnly=true'
 )
 
+# PowerDNS mapping configuration
+#
+# Tell Serveradmin which servertypes represent domains and records and which
+# attributes represent the domain and record information.
+#
+PDNS = {
+    'domain': [{
+        'servertype': 'domain',
+        'id': 'object_id',
+        'name': 'hostname',
+        'master': None,
+        'type': None,  # Default: NATIVE
```

Comment:
Does that mean that if `None` than it will default to `NATIVE`? It's not completely understandable like that. Could also mean, that right now it is set to `None` but default is that it should set to `NATIVE`.
Maybe a wording like `Defaults to NATIVE if None` is more descriptive?

Reply:
Used your wording.